### ISSUE TITLE
A hash conflict was encountered for the role Assignment ID

### ISSUE THREAD

**Issue by jim-minter** [Azure/ARO-RP#670]
Saw this terminal error on an e2e, I guess concurrently creating role assignments for the RP SP on the same vnet in two regions at the same time:

```
########## 🚀 Create ARO Cluster v4-e2e-V31050325 - Using client-id : 29ab2cbe-e3d7-48ba-9aa3-74131787b400 ##########
The behavior of this command has been altered by the following extension: aro
No --pull-secret provided: cluster will not include samples or operators from Red Hat or from certified partners.
A hash conflict was encountered for the role Assignment ID. Please use a new Guid.
```

Please can you recreate the condition and make the error retriable?

**Comment by jim-minter** [comment]
This is surely down to uuid5.

**Comment by jim-minter** [comment]
@petrkotas please can you take this on?  Also please join the Azure GH org so I can assign this to you :-)

**Comment by jim-minter** [comment]
https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=31050325&view=logs&j=d33b4a02-be2f-5011-6e5d-14ea912a015b&t=c3e44864-8e85-535c-cfe2-4790856824f2

**Comment by petrkotas** [comment]
Hi @jim-minter I am working on it. You are right it is related to UUID5.

**Comment by jim-minter** [comment]
This isn't a PUT race - it still happens on retry.

**Comment by petrkotas** [comment]
@jim-minter I do believe this issue has been covered in the merged patch #675.
Therefore this issue can be closed.
